src/expander.wisp

```diff
@@ -264,14 +264,24 @@
   "Thread first macro"
   [& operations]
   (reduce
-   (fn [form operation]
-     (cons (first operation)
-           (cons form (rest operation))))
-   (first operations)
-   (map #(if (list? %) % `(~%))
-        (rest operations))))
+    (fn [form operation]
+      (cons (first operation)
+            (cons form (rest operation))))
+    (first operations)
+    (map #(if (list? %) % `(~%))
+         (rest operations))))
 (install-macro! :-> expand-thread-first)
 
+(defn expand-thread-last
```

Comment:
no test for that?

Reply:
I haven't found a single test for existing macros, so I didn't add any for new ones. Though I guess it wouldn't hurt adding tests for `wisp.expander`…